todo-app/public/index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="theme-color" content="#000000">
+    <link rel="stylesheet" href="style.css">
```

Comment:
This is not the right way to include styles in your `create-react-app` project!

You can read [the docs here](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-a-stylesheet) about how styles should work within `create-react-app` projects.

We can also discuss this in person, but basically there is a relatively complicated build toolchain set up to allow you to make your styles modular, and to be able to import them into your js files in a logical way. The days of having a single massive `index.css` are over, we use modular styles now.

A great example is if you build a component, and then suddenly need to use this component in another project. We can simply copy/paste the entire component directory:

    |- MyComponent
      ---- MyComponent.js
      ---- MyComponent.css

And not have to worry about messy global styles.

Reply:
Okay I get it, it seems tempting to have a small amount of css in each file instead of a huge one, but I wonder if it will become hard if there was like a container component and a contained component and they had some relation in the styling like Flex-box or something.

Reply:
Yeah, that's a fair point. Styling React components is the one thing where there is no real "standard" the community has decided upon. What you're bringing up depends upon your methodology for managing React components in a hierarchical way. [Here's one example](https://medium.com/@yejodido/atomic-components-managing-dynamic-react-components-using-atomic-design-part-1-5f07451f261f), although I don't follow it or know enough to recommend it.

Basically - it's up to you.

My point was that it's an anti-pattern to include your own `index.css` in the `index.html` directly (or really put any CSS in the `public` dir), webpack does this for you. As long as you use the build tools in CRA appropriately, the rest is up to you!